src/Worker.php

```diff
@@ -80,9 +80,11 @@ private function tick($queue)
         }
 
         try {
+            $this->event->acknowledge($message);
```

Comment:
Thoughts on moving this up a few lines: https://github.com/equip/queue/blob/master/src/Worker.php#L76